lib/cmds/opts.ts

```diff
@@ -58,8 +59,11 @@ opts[STANDALONE] = new Option(
 opts[CHROME] =
     new Option(CHROME, 'Install or update chromedriver', 'boolean', ChromeDriver.isDefault);
 opts[GECKO] = new Option(GECKO, 'Install or update geckodriver', 'boolean', GeckoDriver.isDefault);
-opts[IE] = new Option(IE, 'Install or update ie driver', 'boolean', IEDriver.isDefault);
+opts[IE] = new Option(
+    IE, 'Install or update ie driver (defaults downloading 32-bit version)', 'boolean',
+    IEDriver.isDefault);
 opts[IE32] = new Option(IE32, 'Install or update 32-bit ie driver', 'boolean', IEDriver.isDefault);
+opts[IE64] = new Option(IE64, 'Install or update x64 iedriver', 'boolean', IEDriver.isDefault);
```

Comment:
`isDefault` is a boolean to either to download or not download. The arch version is decided elsewhere.